monkey/monkey_island/cc/resources/credentials/stolen_propagation_credentials.py

```diff
@@ -0,0 +1,28 @@
+import logging
+
+from flask import jsonify, request
+
+from common.credentials import Credentials
+from monkey_island.cc.repository import ICredentialsRepository
+from monkey_island.cc.resources.AbstractResource import AbstractResource
+
+logger = logging.getLogger(__name__)
+
+
+class StolenPropagationCredentials(AbstractResource):
+    urls = ["/api/propagation-credentials/stolen"]
+
+    def __init__(self, credentials_repository: ICredentialsRepository):
+        self._credentials_repository = credentials_repository
+
+    def get(self):
+        return jsonify(self._credentials_repository.get_stolen_credentials())
```

Comment:
Does `jsonify()` work as expected? Does it properly handle the enums? Or do we need to do `jsonify([Credentials.to_json(c) for c in self._credentials_repository.get_stolen_credentials()])`